tasks/sep/base-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,341 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0x0fe884546476dDd290eC46318785046ef68a0BA9` (The 1 of 1 `ProxyAdmin` owner Safe)
```

Comment:
Looks like the safe Base uses here is a 3/7. This comment is probably residual from conduit's which was a 1/1 at the time. I would just relace "The 1 of 1 `ProxyAdmin` owner Safe" with "The `ProxyAdmin` owner Safe" (occurs here and below)